app/helpers/email_helper.rb

```diff
@@ -26,10 +26,11 @@ def email_topic_link(topic)
   end
 
   def email_html_template
-    EmailStyle.new.html
-      .sub('%{email_content}', capture { yield })
-      .gsub('%{html_lang}', html_lang)
-      .html_safe
+    tmp = EmailStyle.new.html
+    tmp = tmp.sub('%{email_content}') { capture { yield } }
+    tmp = tmp.gsub('%{html_lang}', html_lang)
+    tmp = tmp.html_safe
+    tmp
```

Comment:
Great find!

Would it be better to just change the `{ capture { yield } }` part? I assume `tmp` stuff was just for debugging purposes?

Reply:
Done